modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/socket/ClustersRepository.java

```diff
@@ -171,32 +183,33 @@ public boolean hasDemo(UUID accId) {
     void cleanupClusterIndex() {
         Collection<UUID> nids = U.nodeIds(ignite.cluster().nodes());
 
-        stream(clusterIdsByBrowser.cache().spliterator(), false)
+        stream(clusterIdsByUser.cache().spliterator(), false)
             .peek(entry -> {
                 Set<ClusterSession> activeClusters =
                     entry.getValue().stream().filter(cluster -> nids.contains(cluster.getNid())).collect(toSet());
 
                 entry.getValue().removeAll(activeClusters);
             })
             .filter(entry -> !entry.getValue().isEmpty())
-            .forEach(entry -> clusterIdsByBrowser.removeAll(entry.getKey(), entry.getValue()));
+            .forEach(entry -> clusterIdsByUser.removeAll(entry.getKey(), entry.getValue()));
     }
 
     /**
-     * Periodically cleanup expired cluster topology.
+     * Periodically cleanup expired cluster topologies.
      */
     @Scheduled(initialDelay = 0, fixedRate = 60_000)
     public void cleanupClusterHistory() {
         txMgr.doInTransaction(() -> {
-            Set<String> clusterIds = stream(this.clusters.cache().spliterator(), false)
+            Set<String> clusterIds = stream(clusters.cache().spliterator(), false)
                 .filter(entry -> entry.getValue().isExpired(DEFAULT_CLUSTER_CLEANUP))
                 .map(Cache.Entry::getKey)
                 .collect(toSet());
 
             if (!F.isEmpty(clusterIds)) {
                 clusters.cache().removeAll(clusterIds);
 
-                log.debug("Failed to receive topology update for clusters: " + clusterIds);
+                if (log.isDebugEnabled())
```

Comment:
Why we should check log level for slf4j logger?

Reply:
Fixed